inference-engine/samples/speech_sample/main.cpp

```diff
@@ -1029,6 +1029,11 @@ int main(int argc, char *argv[]) {
                             continue;
                         }
 
+                        ptrInputBlobs.clear();
+                        for (auto& input : cInputInfo) {
+                            ptrInputBlobs.push_back(inferRequest.inferRequest.GetBlob(input.first));
+                        }
+
```

Comment:
@kbruniec , Will this correct work with option -iname?

Reply:
How to test it?
As you are experienced with this, maybe you could check?

Reply:
Ok, I will check )

Reply:
@admitriev-gna, do we have a test which checks nthreads == 2 together with -iname option?

Reply:
No, we haven't such as test.

Reply:
@kbruniec I created draft PR with supporting -iname and your fix
https://github.com/openvinotoolkit/openvino/pull/4774

Reply:
@admitriev-gna Thanks I am reviewing it.

@admitriev-gna @dorloff  is "-iname" documented somewhere? Do we have a test for iname?

I can not find it on https://docs.openvinotoolkit.org/latest/openvino_inference_engine_samples_speech_sample_README.html